R/ingest-scp.R

```diff
@@ -44,8 +44,6 @@ get_input_combinations <- function(cancer_types, race_options, sex_options, age_
     ),
     stringsAsFactors = FALSE
   )
-
-  return(out)
 }
```

Comment:
The last expression evaluated in an R function is the value that gets returned by that function, so there's not need to use `return()` unless you need to return a value before the end of the function. A toy (bad) example:

```r
odd_or_even <- function(x) {
  if(x %% 2 == 0) {
    return("even")
  }

  "odd"
}
```